app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>

```

Comment:
- [ ] I would recommend updating the code to match with wireframe design and styling.

### Given design

![Screenshot from 2021-12-01 03-39-07](https://user-images.githubusercontent.com/22434660/145859986-a7e77da8-6e30-45f1-a4c4-fbf0077779ed.png)

### Your design

![Screenshot from 2021-12-13 22-56-00](https://user-images.githubusercontent.com/22434660/145859974-f3b8a05b-95fb-4d83-9ca6-391e25c26b6e.png)